src/olympia/accounts/utils.py

```diff
@@ -3,27 +3,18 @@
 from urllib import urlencode
 
 from django.conf import settings
-from django.core.urlresolvers import reverse
 from django.http import HttpResponseRedirect
 from django.utils.http import is_safe_url
 
-import waffle
-
 from olympia.amo.utils import urlparams
 
 
 def login_link(request):
-    if waffle.switch_is_active('fxa-migrated'):
-        return default_fxa_login_url(request)
-    else:
-        return link_with_final_destination(request, reverse('users.login'))
+    return default_fxa_login_url(request)
```

Comment:
Is it feasible to strip out the redirection and either stick the code for `default_fxa_login_url` in here or change references to `login_link` to `default_fxa_login_url` where it's used?